src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto

```diff
@@ -264,6 +268,9 @@ service LibraryService {
     option (google.api.method_signature) = {
       fields: ["name"]
     };
+    option (google.longrunning.operation_types) = {
```

Comment:
Is it ok to not have a `response` for this void RPC method?

Reply:
No.

Reply:
(Use `google.protobuf.Empty`.)

Reply:
Done.